recipes/phosphor/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "phosphor" %}
+{% set version = "0.7.1" %}
+{% set sha256 = "375ad74a0e555192bf21f1c412a0a499f0943eeb9048b55054397434973586cd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://registry.npmjs.org/{{ name }}/-/{{ name }}-{{ version }}.tgz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+  skip: True  # [osx]
```

Comment:
Two questions related to this:

+ Will this be respected? Sounds like from the documentation it may not be: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#architecture-independent-packages
+ What is the reason for skipping OSX?